internals/models.py

```diff
@@ -924,8 +922,8 @@ def getSortingMilestone(feature):
       shipping_features = map(getSortingMilestone, shipping_features)
 
       # First sort by name, then sort by feature milestone (latest first).
-      shipping_features.sort(key=lambda f: f.name, reverse=False)
-      shipping_features.sort(key=lambda f: f._sort_by_milestone, reverse=True)
+      sorted(shipping_features, key=lambda f: f.name, reverse=False)
```

Comment:
The old .sort() method modified the array in-place.  These lines just return a sorted copy without changing the original, so they are no-ops unless we store the result back into the same variable.

Does python 3 not have .sort()?  I thought that it did.

Reply:
shipping_features.sort(key=lambda f: f.name, reverse=False)
AttributeError: 'map' object has no attribute 'sort'

In python 3 map doesn't return a list but an iterator

Reply:
sorry I didn't address a part of your comment. I should reassign the result of `sorted`

Reply:
Ah, so up on line 922 it should be:
shipping_features = list(map(getSortingMilestone, shipping_features))

Then these two lines can go back to using .sort()

Reply:
Don't we want to maintain `shipping_features` as a map?

Reply:
Actually [sorted](https://docs.python.org/3/howto/sorting.html) returns a list so it is the same